packages/frontend-design-poc/src/components/InboxItem/InboxItem.tsx

```diff
@@ -0,0 +1,122 @@
+import { Checkbox } from "@digdir/design-system-react";
+import classNames from "classnames";
+
+import styles from "./inboxItem.module.css";
+
+/**
```

Comment:
Is VSCode able to pick up this comment when using an `<InboxItem>` component? Or does it have to be just above the component definition (on line 66). I haven't tested this so I'm not sure.

If it works the way it is then this is great.